packages/docs/src/components/DocSiteFooter.tsx

```diff
@@ -1,7 +1,128 @@
 import React from 'react';
+import uniqueId from 'lodash/uniqueId';
+import githubUrl from '../helpers/githubUrl';
+
+/**
+ * Privacy setting link under addtional resources triggers modal (styles out of scope)
+ */
+
+const governmentLinks = {
+  'http://www.cms.gov': 'CMS.gov',
+  'http://www.medicare.gov': 'Medicare.gov',
+  'http://www.mymedicare.gov': 'MyMedicare.gov',
+  'http://www.medicaid.gov': 'Medicaid.gov',
+  'http://www.healthcare.gov': 'HealthCare.gov',
+  'http://www.HHS.gov/open': 'HHS.gov',
+};
+
+const resourceLinks = {
+  'https://designsystem.digital.gov/': 'U.S. Web Design System',
+  'http://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/FOIA.html':
+    'Freedom of Information Act',
+  'https://oig.hhs.gov/': 'Inspector General',
+  'http://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/NoFearAct.html': 'No Fear Act',
+  'http://www.medicare.gov/about-us/plain-writing/plain-writing.html': 'Plain writing',
+  'http://www.usa.gov': 'USA.gov',
+  'https://cms.gov/privacy/': 'Privacy policy',
+};
+
+/**
+ * @param {Array[Object]} links
+ * @return {Array[<li>]}
+ */
+function renderLinks(links) {
+  const urls = Object.getOwnPropertyNames(links);
+
+  return urls.map(function (url) {
+    return (
+      <dd key={uniqueId()}>
+        <a href={url} target="_blank" rel="noreferrer">
+          {links[url]}
+        </a>
+      </dd>
+    );
+  });
+}
 
 const DocSiteFooter = () => {
-  return <footer className="c-footer">Placeholder footer content here</footer>;
+  return (
+    <footer className="c-footer">
+      <section className="c-footer__content--top">
```

Comment:
[nitpick] Consider updating this class name to `c-footer__content c-footer__content--top` to be more inline with BEM. 

The current class name makes the most sense if the class `c-footer__content` also existed on this element since `--top` is implying a modification of the `c-footer__content` class.

[In the scss file](https://github.com/CMSgov/design-system/pull/1733/files#diff-c9decc8d17c1f72ac7135030f6d0ba091178b70dc5137580606b8e1bc63d5578R9), it looks like there are styles that apply to both `c-footer__content--bottom` & `c-footer__content--top`. These styles could be instead scoped to the class `c-footer__content`